scotty

```diff
@@ -105,7 +109,14 @@ def create_venv():
 
     if not os.path.isdir(venv_real_dir):
         print_color("\nCreating virtual env in %s..." % Config.get("venv"))
-        run_commands(["%s %s" % (str(Config.get("virtualenv")), str(venv_real_dir))])
+        if run_commands(["%s %s" % (str(Config.get("virtualenv")), str(venv_real_dir))], False):
+            print_color("\nDefault virtualenv path doesn't match, trying to find it manually...")
+            venvpath = Config.findVenv()
+            if venvpath:
+                run_commands(["%s %s" % (str(venvpath), str(venv_real_dir))])
+            else:
+                print_color("\nDidn't find file, did virtualenv install correctly? :/")
```

Comment:
This isn't handled in start_venv, maybe in config there should be a default get venv function, it tries the basic ones in the config first and then at last attempts tries to find it. That way every place that uses it benefits from finding it. And maybe even cache the result (Either in memory so it runs it once or edit the config.json)

Reply:
The main idea was to make it so that we didn't edit the file for everyone with a different path, and wind up with one person's path conflicting with another person's. That's kind of the whole reason the default path doesn't just work. Either that, or depend on a separate config file that's ignored by git, and is almost identical except for the path for virtualenv... but that seems inefficient.

With the way the code works currently, the command is run off of the path found in the config file by default, no function, just structure referencing.
We could make a conditional execution to find the path by default if someone asks for the 'virtualenv' attribute, and we wouldn't have both attempts, although the `whereis` check should hopefully either match or be the correct version.... that's actually a fair idea.